### PR TITLE
Added Implementation of Prime Numbers using JS

### LINKED ISSUE #1567: Writing prime number algorithm  in JS
@raghav-dalmia I am a participant in GSSoC, and I would like to work on the implementation of a prime number in JS.

### PR DESCRIPTION
**Pull request by abhu-A-J**
Fixes #1567  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- The Prime Numbers folder had no implementation in JavaScript so added the necessary script.

#### Languages Used:
- JavaScript

#### Files Added:
- New File added to Prime_Numbers folder 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by abhu-A-J** [comment]
@raghav-dalmia @asha15 kindly review the PR, so I can move one to more harder algorithms. If any changes required kindly revert back. Thanks.

**Comment by abhu-A-J** [comment]
@singh-shreya6 I have made the changes required.

File: Prime_Number/Prime_Number.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//Description: Function to find if a given number is prime or not
+//Expected Output: returns true or false
+
+const isPrime = (num) => {
+  //Numbers less than 2 are not prime numbers (including negatives)
+  if (num < 2) {
```
Comment (singh-shreya6) [inline Prime_Number/Prime_Number.js]: please add 4 space indentation here as well as it is inside the block for function

File: Prime_Number/Prime_Number.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      return false;
+  }
+  //start finding factors for num , if only 2 factors then it's prime, else not
+  let i = 1, factors = 0;
```
Comment (singh-shreya6) [inline Prime_Number/Prime_Number.js]: same.

**Comment by abhu-A-J** [comment]
@singh-shreya6  how about now? that seems correct??

**Comment by raghav-dalmia** [comment]
Looks good to me. I am merging this.

**Comment by abhu-A-J** [comment]
@raghav-dalmia thank you!

File: Prime_Number/Prime_Number.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+//Description: Function to find if a given number is prime or not
```
Comment (singh-shreya6) [inline Prime_Number/Prime_Number.js:1]: Please change this to a single multi line comment

File: Prime_Number/Prime_Number.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+const isPrime = (num) => {
+    //Numbers less than 2 are not prime numbers (including negatives)
+    if (num < 2) {
+        return false;
+    }
+    //start finding factors for num , if only 2 factors then it's prime, else not
+    let i = 1, factors = 0;
+    while (i <= num) {
+        let rem = num % i; //check the remiander is zero or not
```
Comment (singh-shreya6) [inline Prime_Number/Prime_Number.js:12]: Please fix typo in word remainder in comment

File: Prime_Number/Prime_Number.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    }
+    if (factors === 2) { //exactly two factors makes it prime
+        return true;
+    }
+    else {  //else it not a prime number
+        return false;
+    }
+}
+
+//I/P and O/P Examples
```
Comment (singh-shreya6) [inline Prime_Number/Prime_Number.js:26]: Can you remove the true/false from comment and add an output section at end of file for this inputs as multi line comment.

**Pull request merged**